showup/views.py

```diff
@@ -7,7 +7,29 @@
 from .filters import ConcertFilter
 
 
+@login_required
```

Comment:
This is incorrect. Take a look at the following screenshot from `develop`:

<img width="1280" alt="Screen Shot 2019-12-10 at 3 35 24 PM" src="https://user-images.githubusercontent.com/15850452/70566776-b70c4100-1b62-11ea-8e46-3b9bc8d13aa7.png">

Whoever is viewing this page is not currently logged in, yet viewing `/`.